pkg/syncer/util.go

```diff
@@ -127,6 +127,7 @@ func IsValidVolume(ctx context.Context, volume v1.Volume, pod *v1.Pod, metadataS
 // if volumeIds is empty, then all volumes from CNS will be retrieved by pagination
 func getQueryResults(ctx context.Context, volumeIds []cnstypes.CnsVolumeId, clusterID string, volumeManager volumes.Manager) ([]*cnstypes.CnsQueryResult, error) {
 	log := logger.GetLogger(ctx)
+	log.Debugf("getQueryResults is called with volumeIds %v for clusterID %s", volumeIds, clusterID)
```

Comment:
Is the function `getQueryResults` only invoked by `fullsync` ?
If yes, can you add the prefix `"FullSync: "` in all the logging statements in this function.

Reply:
Done.